src/mon/OSDMonitor.cc

```diff
@@ -10986,6 +10986,17 @@ bool OSDMonitor::prepare_command_impl(MonOpRequestRef op,
       goto reply;
     }
 
+    if (expected_num_objects == 0 &&
+	cct->_conf->osd_objectstore == "filestore" &&
+	cct->_conf->filestore_merge_threshold < 0) {
+      if (pg_num >= 1024 || pg_num / osdmap.get_num_osds() >= 50) {
```

Comment:
Did we change the num of pgs per osd threshold from 100 to 50?

Reply:
Didn't we? I thought we did in the last conversation we had. If you think we should keep at 100, I'm ok with that.